docs/reference/ml/anomaly-detection/apis/find-file-structure.asciidoc

```diff
@@ -206,6 +210,20 @@ structure finder does not consider by default.
 If this parameter is not specified, the structure finder chooses the best
 format from a built-in set.
 
+Timeformat examples:
```

Comment:
> Timeformat examples

I think rather than having this heading, we should provide a bit of introductory text.  Something like "The following table provides the appropriate `timeformat` values for some example timestamps:"